packages/ember-css-modules/lib/resolve-path.js

```diff
@@ -49,7 +49,12 @@ function resolveLocalPath(importPath, fromFile, options) {
 function resolveExternalPath(importPath, originalPath, fromFile, options) {
   let baseIndex = importPath[0] === '@' ? importPath.indexOf('/') + 1 : 0;
   let addonName = importPath.substring(0, importPath.indexOf('/', baseIndex));
-  let addon = options.parent.addons.find(addon => addon.name === addonName);
+  let addon = options.parent.addons.find((addon) => {
+    if (addon.moduleName) {
+      return addon.moduleName() === addonName;
+    }
+    return addon.name === addonName;
```

Comment:
This looks like it might be a breaking change. It seems like we could avoid that with something like this:

```suggestion
    if (addon.moduleName) {
      if (addon.moduleName() === addonName) { return true; }
      if (addon.name === addonName) {
        // issue deprecation warning about needing to use the module name instead
        return true;
      }
    } else {
      return addon.name === addonName;
    }
```

Reply:
Good catch - this makes sense to me. I'll test this out and update.

Reply:
Had to make this check a little further up the call stack, but I've updated to fall back to checking the old package name if moduleName misses.